internal/conditions/tools/BUILD.bazel

```diff
@@ -0,0 +1,10 @@
+package(default_visibility = [
+    "//visibility:public",
+])
+
+config_setting(
+    name = "coverage",
+    values = {
+        "collect_code_coverage": "true",
+    },
+)
```

Comment:
this may need to use [`ctx.configuration.coverage_enabled`](https://docs.bazel.build/versions/2.2.0/skylark/lib/configuration.html#coverage_enabled) instead

Reply:
That would be better as it takes `--instrumentation_filter` into account.